docs/book/common_code/cluster_controller.md

```diff
@@ -0,0 +1,156 @@
+
+# Cluster Resources
+
+A `Cluster` represents the global configuration of a Kubernetes cluster.
+
+{% panel style="info", title="Important" %}
+To see the actual and current definitions see the [source](#cluster_source).
+{% endpanel %}
+
+{% method %}
+## Cluster
+
+Cluster has 4 fields:
+
+`Spec` contains the desired cluster state specified by the object. While much
```

Comment:
This could be a little clearer if the order here matched the order they appeared in the struct below. I think a personal preference is to put the struct first then the explanation after, but I think this ordering works as well

Reply:
Thinking more about this, I am not sure I agree. By placing the `Spec` and `Status` descriptions first aren't we are emphasizing the portion of the API which matters most to the implementor?

Reply:
i can see it both ways, totally fine to keep it as is